src/main/java/io/vertx/kafka/client/consumer/impl/KafkaConsumerImpl.java

```diff
@@ -389,7 +389,11 @@ public void committed(TopicPartition topicPartition, Handler<AsyncResult<OffsetA
       if (done.succeeded()) {
         // TODO: use Helper class and stream approach
         List<PartitionInfo> partitions = new ArrayList<>();
-        for (org.apache.kafka.common.PartitionInfo kafkaPartitionInfo: done.result()) {
+        List<org.apache.kafka.common.PartitionInfo> partitionInfos = done.result();
+        if (partitionInfos == null){
+          partitionInfos = new ArrayList<>();
```

Comment:
Instead of returning an empty list of partitions, shouldn't we just fail the handler with a proper message saying, ```the topic does not exist and the auto-create topic is disabled```
?

Reply:
I have pushed a new commit. Now, this fails when `partitionInfos` is null.